stage_descriptions/geospatial_2_wq4.md

```diff
@@ -0,0 +1,34 @@
+In this stage, you'll add support for inserting multiple locations in one `GEOADD` command.
```

Comment:
Let's ignore this from scope and stick to adding one location at once, similar to what we did with the `SUBSCRIBE` command. It also makes logs simpler when we show one place per line everywhere.

We can keep the test for retrieving multiple locations though, the response type there is an array so it'll be weird for users to hardcode an array with a single element when implementing. The syntax is also shorter than this (`GEOPOS key loc_1 loc_2`)